src/tools/build_helper/src/ci.rs

```diff
@@ -36,25 +36,26 @@ impl CiEnv {
 }
 
 pub mod gha {
-    use std::sync::atomic::{AtomicBool, Ordering};
+    use std::sync::{Mutex, OnceLock};
 
-    static GROUP_ACTIVE: AtomicBool = AtomicBool::new(false);
+    static ACTIVE_GROUPS: OnceLock<Mutex<Vec<String>>> = OnceLock::new();
```

Comment:
Do we need a OnceLock here? [Mutex::new](https://doc.rust-lang.org/nightly/std/sync/struct.Mutex.html#method.new) is const?

Reply:
Good point, thanks!